sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -128,6 +128,14 @@ def set_power_override(self, power_override, power_set):
     def get_eeprom_path(self):
         raise NotImplementedError
 
+    def get_lpmode(self):
```

Comment:
the return type should match with the return type mentioned in sfp_base.py for this api

Reply:
the above change will help us to check whether API is implemented or not. Incase False is returned, it'd be difficult to catch where False is returned(whether the module is not in LP mode/ API is not defined).